scripts/rhel_validation/check_lis_modules.sh

```diff
@@ -21,8 +21,8 @@ main() {
         mod_found=$?
         if [[ $mod_found -eq 0 ]];then
             mod_ver="$(modinfo $module | grep -w version)"
-            mod_ver=${mod_ver#*:}
-            echo "${module}: ${mod_ver}"
+	    [[ -z $mod_ver ]] && mod_ver="in-box driver" || mod_ver=${mod_ver#*:}
```

Comment:
fix indentation, use only spaces, no TABs.

Reply:
Fixed

Reply:
changes are missing, please check you pushed them

Reply:
Fixed.